windows_containers/windows-containers-release-notes-10-16-x.adoc

```diff
@@ -0,0 +1,101 @@
+
+[id="wmco-10-16-0-operator-kube"]
+==== Kubernetes upgrade
+
+The WMCO now uses Kubernetes 1.29.
+
+[id="wmco-10-16-0-bug-fixes"]
+=== Bug fixes
+
+Because the WICD service account was missing a required secret, the WMCO was unable to properly configure Windows nodes in a Nutanix cluster. With this fix, the WMCO creates a long-lived token secret for the WICD service account. As a result, the WMCO is able to configure a Windows node on Nutanix. (link:https://issues.redhat.com/browse/OCPBUGS-22680[*OCPBUGS-22680*])
+
+Previously, the WMCO performed a sanitization step that incorrectly replaced commas with semicolons in the user's cluster-wide proxy configuration. This caused Windows to ignore the values set in the `noProxy` environment variable. As a consequence, the WMCO incorrectly sent traffic through the proxy for the endpoints specified in the `no-proxy` parameter. With this fix, the sanitization step that replaced commas with semicolons was removed. As a result, web requests from a Windows node to a cluster-internal endpoint or an endpoint that exists in the `no-proxy` parameter does not go through the proxy. (link:https://issues.redhat.com/browse/OCPBUGS-24264[*OCPBUGS-24264*])
+
+// Copied from 4.15
+Previously, because of bad logic in the networking configuration script, the WMCO was incorrectly reading carriage returns in the containderd CNI configuration file as changes, and identified the file as modified. This caused the CNI configuration to be unnecessarily reloaded, potentially resulting in container restarts and brief network outages. With this fix, the WMCO now reloads the CNI configuration only when the CNI configuration is actually modified. (link:https://issues.redhat.com/browse/OCPBUGS-2887[*OCPBUGS-2887*])
+
+// Copied from 4.12
+Previously, because of routing issues present in Windows Server 2019, under certain conditions and after more than one hour of running time, workloads on Windows Server 2019 could have experienced packet loss when communicating with other containers in the cluster. This fix enables Direct Server Return (DSR) routing within kube-proxy. As a result, DSR now causes request and response traffic to use a different network path, circumventing the bug within Windows Server 2019. (link:https://issues.redhat.com/browse/OCPBUGS-26761[*OCPBUGS-26761*])
+
+// Copied from 4.15
+Previously, the kubelet on Windows nodes was unable to authenticate with private Elastic Container Registries (ECR) registries. Because of this error, the kubelet was not able to pull images from these registries. With this fix, the kubelet is able to pull images from these registries as expected. (link:https://issues.redhat.com/browse/OCPBUGS-26602[*OCPBUGS-26602*])
```

Comment:
```suggestion
Previously, the kubelet on Windows nodes was unable to authenticate with private Elastic Container Registries (ECR) registries. Because of this error, the kubelet was not able to pull images from these registries. With this fix, the kubelet is able to pull images from these registries as expected. (link:https://issues.redhat.com/browse/OCPBUGS-26602[*OCPBUGS-26602*])
```
- I'm wondering if there's a way to reword to avoid saying `...Registries (ECR) registries`?